### ISSUE TITLE
The use of split in the loss function? What is it for?

### ISSUE THREAD

**Issue by jahdiel** [Microsoft/human-pose-estimation.pytorch#99]
This code is in core/loss.py:
`heatmaps_pred = output.reshape((batch_size, num_joints, -1)).split(1, 1)`

I was just wondering what is the `split(1,1)` for? I From my understanding it splits the tensor into equally sized tensors along the 1 axis, but I am not understanding why split into tensors of size 1? Hope somebody can help. Thanks

**Comment by sjiang17** [comment]
I believe the author did it just for the convenience of using target weight. No other particular reasons.
